specification/hybridconnectivity/resource-manager/Microsoft.HybridConnectivity/stable/2023-03-15/hybridconnectivity.json

```diff
@@ -0,0 +1,1093 @@
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "HybridConnectivityEndpointsServiceconfigurationsGetSSH": {
+            "$ref": "./examples/ServiceconfigurationsGetSSH.json"
+          },
+          "HybridConnectivityEndpointsServiceconfigurationsGetWAC": {
+            "$ref": "./examples/ServiceconfigurationsGetWAC.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceUriParameter"
+          },
+          {
+            "$ref": "#/parameters/EndpointNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceConfigurationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Service details",
+            "schema": {
+              "$ref": "#/definitions/ServiceConfigurationResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "Serviceconfigurations"
+        ],
+        "description": "Create a service in serviceConfiguration for the endpoint resource.",
+        "operationId": "Serviceconfigurations_Create",
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "ServiceconfigurationsPutSSH": {
+            "$ref": "./examples/ServiceconfigurationsPutSSH.json"
+          },
+          "ServiceconfigurationsPutWAC": {
+            "$ref": "./examples/ServiceconfigurationsPutWAC.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceUriParameter"
+          },
+          {
+            "$ref": "#/parameters/EndpointNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceConfigurationNameParameter"
+          },
+          {
+            "name": "ServiceConfigurationResource",
+            "description": "Service details",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServiceConfigurationResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ServiceConfiguration Resource details.",
+            "schema": {
+              "$ref": "#/definitions/ServiceConfigurationResource"
+            }
+          },
```

Comment:
When PUT is a 'CreateOrUpdate' semantic, it must declare both response codes: 201 and 200

Reply:
PUT  API in our case will only be creating the resource. For updates, we recommend our users to use PATCH API by our design.